shapely/geometry/linestring.py

```diff
@@ -23,6 +23,11 @@ class LineString(BaseGeometry):
         an array-like with shape (N, 2) or (N, 3).
         Also can be a sequence of Point objects, or combination of both.
 
+    See Also
+    --------
+    linestrings : Create an array of linestrings.
```

Comment:
```suggestion
    linestrings : Create an array of LineStrings.
```

I was going to say that I think we mostly use the camel case form (at least in this docstring), but definitely not consistently because the docstrings of the creation functions we are linking to indeed use the lower case version ..

Reply:
Yes, it is indeed not very consistent in general, and I indeed just reused the docstring of the creation function.

Maybe one or the other approach could be choosen and then make a PR to change it everywhere to that?

Reply:
@theroggy @jorisvandenbossche In my original manual, I tried to use "polygon" (for example) when talking abstractly about what a function does, like in the first paragraph of a docstring. But then I tried to always use "Polygon" when talking about the implementation. I didn't always succeed, but this was the intention.

Python's own docs are written this way. See https://docs.python.org/3/library/stdtypes.html#string-methods for example, and `str.split()` specifically:

> Return a list of the words in the string, using sep as the delimiter string.

It says "string" and not "str", because it is emphasizing the concepts, not the implementations. So, we can do the same thing. We can use "line string" when we want to highlight the concept and "LineString" when we want to emphasize the implementation.

Reply:
That sounds as a good general policy to me! (although of course it will then still always be a bit ambiguous which of the two concepts you are talking about ;))

Here, we are creating the actual objects of class LineString. So would we then say `Create an array of LineStrings.`?

Reply:
I agree it is a good policy in theory, but indeed I'm afraid it will require some serious thinking and possibly discussing for each and every instance of these words...